app/src/main/java/org/fossasia/openevent/app/common/binding/BindingAdapters.java

```diff
@@ -17,6 +18,11 @@
 
 public class BindingAdapters {
 
+    @BindingConversion
+    public static String longToStr(Long value) {
+        return value != null ? String.valueOf(value) : "";
```

Comment:
Suppose a model does not include a primitive long, but the `Long` class, and it is missing from the API response. Or just if someone wanted to make the app crash and created a null long and bind it in the layout